pkg/network/netbinding/netbinding.go

```diff
@@ -48,11 +50,7 @@ func NetBindingPluginSidecarList(vmi *v1.VirtualMachineInstance, config *v1.Kube
 		}
 	}
 
-	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
-		if iface.Slirp != nil {
-			pluginSidecars = append(pluginSidecars, SlirpNetBindingPluginSidecar(vmi, config, recorder))
-		}
-	}
+	pluginSidecars = append(pluginSidecars, SlirpNetBindingPluginSidecar(vmi, config, recorder)...)
```

Comment:
I would change `SlirpNetBindingPluginSidecar` to `SlirpNetBindingPluginImage` and add it to `bindingByName`.
This approach will assure the is only one slirp sidecar, no matter how many interfaces used Slirp plugin or old binding method.

Please also add tests to cover this mutliple Slirp interfaces scenario (some with slirp plugin and some with old slirp binding).